sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -92,6 +93,7 @@ internal class ViewAnnotationManagerImpl(
     currentlyDrawnViewIdSet.clear()
     annotationMap.clear()
     idLookupMap.clear()
+    annotationsMap.clear()
```

Comment:
I see clearing whole map on `removeAllViewAnnotations` but can't see removing an item when `removeViewAnnotation` is called

Reply:
good catch 💎